hack/debug.sh

```diff
@@ -179,7 +200,7 @@ do_pod_to_pod_connectivity_check () {
     echo_and_eval ovs-appctl ofproto/trace br0 "in_port=${other_pod_port},reg0=${other_pod_vnid},ip,nw_src=${other_pod_addr},nw_dst=${base_pod_addr},dl_dst=${base_pod_ether}"
     echo ""
 
-    if nsenter -n -t $base_pod_pid ping -c 1 -W 2 $other_pod_addr >& /dev/null; then
+    if nsenter -n -t $base_pod_pid  ping -c 1 -W 2 $other_pod_addr  &> /dev/null; then
```

Comment:
ah, "to make the command being run more obvious"; we should probably use `--` between the nsenter args and the command anyway